eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -148,7 +148,7 @@ object Commitments {
 
     val htlcValueInFlight = outgoingHtlcs.map(_.add.amountMsat).sum
     // we must use unsigned comparison here because 'maxHtlcValueInFlightMsat' is effectively an uint64 and can exceed the capacity of MilliSatoshi class
-    if (htlcValueInFlight.compareUnsigned(commitments1.remoteParams.maxHtlcValueInFlightMsat) > 0) {
+    if (commitments1.remoteParams.maxHtlcValueInFlightMsat.compareUnsigned(htlcValueInFlight) < 0) {
```

Comment:
If you implement `<` you can write `commitments1.remoteParams.maxHtlcValueInFlightMsat < htlcValueInFlight` which is much better :)